mailpoet/lib/Automation/Engine/WordPress.php

```diff
@@ -96,4 +118,38 @@ public function getAvatarUrl($idOrEmail, $args = null) {
   public function getOption(string $optionName, $default = false) {
     return get_option($optionName, $default);
   }
+
+  public function getCommentStatuses(): array {
+    return get_comment_statuses();
+  }
+
+  /**
+   * @return array<int,int|string|WP_Term>|string|WP_Error
+   */
+  public function wpGetPostTerms(int $postId, string $taxonomy, array $args = []) {
+    return wp_get_post_terms($postId, $taxonomy, $args);
+  }
+
+  /**
+   * @param int|\WP_Comment $comment
+   * @return false|string
+   */
+  public function wpGetCommentStatus($comment) {
+    return wp_get_comment_status($comment);
+  }
+
+  /**
+   * @return string[]|\WP_Post_Type[]
+   */
+  public function getPostTypes(array $args = [], string $output = 'names', string $operator = 'and'): array {
+    return get_post_types($args, $output, $operator);
+  }
+
+  public function postTypeSupports(string $type, string $feature): bool {
+    return post_type_supports($type, $feature);
+  }
+
+  public function getTaxonomies(array $args = [], string $output = 'names', string $operator = 'AND'): array {
+    return get_taxonomies($args, $output, $operator);
+  }
```

Comment:
Some of the new methods have PHPDocs, some don't. Do they have PHPDoc in WP? For IDE experience and static analysis, it would be best to copy all the types.

Reply:
8b54142da adds the return types for two methods to specify it is `string[]` instead of just `array`. I think, we do not need to add doctypes where we have them defined in the method definition.

E.g. I don't think we need it in this case:
```
public function postTypeSupports(string $type, string $feature): bool
```

But I agree that `: array` should be specified.